### DISCUSSION TITLE
Ledger address

### DISCUSSION THREAD

**Discussion by xiaoyuaiheshui** [filecoin-project/lotus#7925] [General Q&A] [answered]
Can not set-owner to a Ledger address, ERROR: mpool push: failed to sign message: Unexpected method

**Comment by SgtCoin** [comment]
@xiaoyuaiheshui Do you have ledger in expert mode? Also, something to look into. https://github.com/Zondax/ledger-filecoin/issues/65

**Comment by SgtCoin** [comment, marked as answer]
Archiving due to no response.
